### PR TITLE
Fixed some issues

### PR DESCRIPTION
**Pull request by Shariefie**
Check my code please

### CODE REVIEW

File: Core/Src/logic_layer.c [hunk 1]
```diff
@@ -29,8 +29,8 @@ uint8_t font_style(char command[])
 			return i;
 		}
 	}
-	softonErrorHandler(ERROR_FONTSTYLE_NOT_FOUND);
-	return 0;
+	//softonErrorHandler(ERROR_FONTSTYLE_OUT_OF_RANGE);
```
Comment (bestehustudentEU) [inline Core/Src/logic_layer.c]: delete this! not comment

**Review by bestehustudentEU — COMMENTED** [review]
just one thing

**Review by Tjerk258 — APPROVED** [review]
Ziet er goed uit.

**Review by bestehustudentEU — APPROVED** [review]
!

**Pull request merged**